### PR TITLE
polls: add new signal which is called after a user voted on a poll

### PR DESCRIPTION
**Pull request by goapunk**
**Tasks**
- [ ] PR name contains story or task reference
- [ ] Documentation (docs and inline)
- [x] Tests (including n+1 and django_assert_num_queries where applicable)
- [x] Changelog

### CODE REVIEW

File: adhocracy4/polls/signals.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+from django.dispatch import Signal
+
+poll_voted = Signal()
```
Comment (m4ra) [inline adhocracy4/polls/signals.py:3]: Can you add here the arguments that are needed for the signal.

File: adhocracy4/polls/api.py [hunk 2]
```diff
@@ -178,6 +179,8 @@ def vote(self, request, pk):
             self.save_vote(question, vote_data, creator, content_id)
 
         poll = self.get_object()
+        print("##########################################")
+        poll_voted.send(sender=None, poll=poll, creator=creator, content_id=content_id)
```
Comment (m4ra) [inline adhocracy4/polls/api.py:183]: sender is the poll view, I have added it as the `self.__class__` in my PR for mB  kiezradar.

Comment (m4ra) [reply adhocracy4/polls/api.py:183]: and the print statement can go I believe :)

**Review by m4ra — CHANGES_REQUESTED** [review]
a few things

File: adhocracy4/polls/signals.py (hunk 1 above)
Comment (goapunk) [reply adhocracy4/polls/signals.py:9]: added a docstring

File: adhocracy4/polls/api.py (hunk 2 above)
Comment (goapunk) [reply adhocracy4/polls/api.py]: thanks, added

**Review by m4ra — COMMENTED** [review]
an extra thing for the tests

**Review by m4ra — APPROVED** [review]
Nice, thanks!

**Pull request merged**